cli/Cargo.toml

```diff
@@ -87,6 +86,7 @@ solana-transaction-error = { workspace = true }
 solana-transaction-status = { workspace = true }
 solana-udp-client = { workspace = true }
 solana-version = { workspace = true }
+solana-vote = { workspace = true }
```

Comment:
Should we just rename this to alpenglow-vote so that we know this is definitely not TowerBFT vote?